stage0/run.go

```diff
@@ -443,12 +455,12 @@ func prepareAppImage(cfg PrepareConfig, appName types.ACName, img types.Hash, cd
 			}
 		}
 
-		if err := ioutil.WriteFile(common.AppTreeStoreIDPath(cdir, appName), []byte(treeStoreID), 0700); err != nil {
+		if err := ioutil.WriteFile(common.AppTreeStoreIDPath(cdir, appName), []byte(treeStoreID), defaultRegularFilePerm); err != nil {
 			return fmt.Errorf("error writing app treeStoreID: %v", err)
 		}
 	} else {
 		ad := common.AppPath(cdir, appName)
-		err := os.MkdirAll(ad, 0755)
+		err := os.MkdirAll(ad, defaultRegularDirPerm)
```

Comment:
`/opt/stage2/busybox`: `0755` is changed to `0750` here.

When using user namespaces, It has the rights `drwxr-x---.  3 root      root`, i.e. "other" does not have permissions on it. `prepare-app` (running inside the container) fails to mount it because it is not running as the same `root` user as on the host.

```
         Starting Prepare minimum environment for chrooted applications...
Error: Make / a mount point failed: Permission denied
[FAILED] Failed to start Prepare minimum environment for chrooted applications.
See 'systemctl status prepare-app@opt-stage2-busybox-rootfs.service' for details.
[DEPEND] Dependency failed for Application=busybox Image=busybox.
```